components/post_view/post_list/index.jsx

```diff
@@ -0,0 +1,448 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import LoadingScreen from 'components/loading_screen.jsx';
+import CreateChannelIntroMessage from 'components/post_view/channel_intro_message';
+import Constants from 'utils/constants.jsx';
+import {getNewestPostIdFromPosts, getOldestPostIdFromPosts} from 'utils/post_utils';
+
+import PostList from './post_list';
+
+const POSTS_PER_PAGE = Constants.POST_CHUNK_SIZE / 2;
+
+export default class PostListWrapper extends React.PureComponent {
```

Comment:
Finally this component has it's own folder :tada:.

Just a question, why a wrapper and not our typical container pattern?

Reply:
We already have a connected component and the props are distributed for both the components. Render cycle in both the components will stay the same even with another connected component. So, I don't see this as a pattern that should hurt us in terms of performance. Personally i favor normal pattern if there isn't any benefit. I will wait to hear harrison thoughts on this.

Reply:
@hmhealey We have two react components here so, i changed it to post_view and post_list but both of these dont have folders now. They are in the root folder of post_view